taskfiles/rp.yml

```diff
@@ -134,6 +134,7 @@ tasks:
     deps:
     - set-aio-max
     - start-compose-cluster
+    - :wasm:publish-local
```

Comment:
@ivotron is this ok to place here?

Reply:
it will conflict with the internal `v-rp:run-ducktape-tests`. Or rather, having it this way means that this is both installing node from source and from pre-compiled binaries, with the `wasm:publish-local` step invoking the precompiled node, and the rest using 3rd-party node.

The `wasm:*` and `v-wasm:*` tasks can be refactored in such a way that they don't mix nodejs installations.

Reply:
@ivotron what needs to be done here are we waiting on additional changes on your side?

Reply:
this is actually not a blocker. it’s just that the version of nodejs built by 3rd-party and the one installed from pre-compiled binaries can get out of sync. And also that it'd be increasing the time it takes for CI to run; not much, just the time it takes to download node

Reply:
the other thing that can be done to avoid this problem of having two node installations:
 1. put this in vtools instead, so that this is `v-wasm:publish-local`.
 2. add `v-rp:build` as a dependency of `v-wasm:publish-local` (as that installs 3rd-party nodejs).
 2. add `v-wasm:publish-local` as a dependency of `v-ducktape:run-tests`

Reply:
@ivotron created this issue, will get to it asap https://github.com/vectorizedio/redpanda/issues/1595